guide/blueprints/multiple-services.md

```diff
@@ -41,8 +41,11 @@ Here there are a few things going on:
 * We've added a second service, which will be the database;
   you'll note the database has been configured to run a custom setup script
 * We've injected the URL of the second service into the appserver as a Java system property
-  (so our app knows where to find the database) 
-
+  (so our app knows where to find the database)
+* We've used externalized config to keep secret information out of the blueprint;
+  this is loaded at runtime from an [externalized config provider](../ops/externalized-config.html),
+  such as a remote credentials store
```

Comment:
Missing full stop.

Reply:
style issue - omitting full stops on single-sentence bullet points is common, don't feel strongly but should ideally be conisstent w two bullet poitns above (and elsewhere), which it is